Changed the leftover CD terms

iherman opened the pull request:
This corresponds to issue #132


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/cid/pull/134.html" title="Last updated on Dec 13, 2024, 1:28 PM UTC (6a39670)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/cid/134/75dd250...6a39670.html" title="Last updated on Dec 13, 2024, 1:28 PM UTC (6a39670)">Diff</a>

dlongley commented on index.html:
```diff
@@ -2695,12 +2695,12 @@ <h3>Processing Errors</h3>
 [[[#retrieve-verification-method]]].
           </dd>
-          <dt id="INVALID_CONTROLLER_DOCUMENT_ID">INVALID_CONTROLLER_DOCUMENT_ID (-22)</dt>
+          <dt id="INVALID_CONTROLLED_IDENTIFIER_DOCUMENT_ID">INVALID_CONTROLLED_IDENTIFIER_DOCUMENT_ID (-22)</dt>
```
Note: I think we removed integers for the errors in the other specs, if so, we should be consistent and remove those here as well. cc: @msporny

iherman replied:
I agree, but maybe cleaner if it is done in a separate PR after this one is merged.

iherman commented:
The issue was discussed in a [meeting](https://www.w3.org/2017/vc/WG/Meetings/Minutes/2024-12-18-vcwg#section2-4) on 2024-12-18

- no resolutions were taken

<details><summary><i>View the transcript</i></summary>

#### 2.4. Changed the leftover CD terms (pr cid#134)

_See github pull request [cid#134](https://github.com/w3c/cid/pull/134)._




**Ivan Herman:** there are a bunch of places where the controller document term was still used, and I changed them to the new name, nothing earthshaking. One thing which is there, for example, the old error message names are "Invalid Controller Document", and that kind of thing.  

**Brent Zundel:** This sounds very straightforward, I encourage folks to review it if they like, I anticipate this PR will be merged by the end of the week.  
… With these decisions we will have addressed all the open PRs. We plan to close issue 115, and we are fully editorial with our remaining issues, even including trying to address 119, as that won't introduce normative language.  
… I believe that after the PRs are processed as we have discussed the document is in a stable place. If you disagree please speak up. With that we could entertain a proposal to move to CR.  

</details>

msporny commented:
Editorial, multiple reviews, no changes requested, no objections, merging.
